torchgeo/models/resnet.py

```diff
@@ -170,6 +170,19 @@ class ResNet50_Weights(WeightsEnum):  # type: ignore[misc]
         },
     )
 
+    FMOW_RGB_GASSL = Weights(
+        url="https://huggingface.co/torchgeo/resnet50_fmow_rgb_gassl/resolve/main/resnet50_fmow_rgb_gassl-44b4461b.pth",  # noqa: E501
```

Comment:
This is the MD5 checksum. torch.hub.download_url uses the SHA256 checksum. Can you change it to da43d987? You'll also need to rename the file on HF.